configs/ecospheres/config.yaml

```diff
@@ -195,11 +195,11 @@ website:
 
 organizations: https://raw.githubusercontent.com/ecolabdata/ecospheres-universe/refs/heads/main/dist/organizations-demo.json
 
-# Configuration format for filters:
+# Configuration format for pages:
 #
-# filters:
-#   {object_type}:               # object type to filter on
-#     universe_query: {dict}     # used to define a "universe" on a given {object_type}, through query args
+# pages:
+#   {page_id}:                   # page identifier
+#     universe_query: {dict}     # used to define a "universe" on a given {page_id}, through query args
 #     tag_prefix: {string}       # prefix for tags on data.gouv.fr
 #     title: {string}            # title for the list page
```

Comment:
Could `title` and/or `breadcrumb_title` take over (or fetch from) the [menu config](https://github.com/opendatateam/udata-front-kit/blob/a7e6b1f0cc871e4ca88ceb91b5c541e8178d699b/configs/ecospheres/config.yaml#L96)?
Probably another PR, just flagging here.

Reply:
Yeah that would be nice, I'll think about it.